### PR TITLE
[config]: add syslog messages to config load_minigraph/reload

### PR DESCRIPTION
**Pull request by lguohan**
add stop/restart services messages in syslog

Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- What I did**
add syslog messages to config load_minigraph/reload

help future debugging

**- How I did it**

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -612,6 +612,7 @@ def reload(filename, yes, load_sysinfo):
             cfg_hwsku = cfg_hwsku.strip()
 
     #Stop services before config push
+    log_info("'reload' stop services...")
```
Comment (jleveque) [inline config/main.py]: Suggest: s/stop/stopping/

File: config/main.py [hunk 2]
```diff
@@ -637,6 +638,7 @@ def reload(filename, yes, load_sysinfo):
     # We first run "systemctl reset-failed" to remove the "failed"
     # status from all services before we attempt to restart them
     _reset_failed_services()
+    log_info("'reload' restart services...")
```
Comment (jleveque) [inline config/main.py]: Suggest: s/restart/restarting/

File: config/main.py [hunk 3]
```diff
@@ -708,6 +711,7 @@ def load_minigraph():
     # status from all services before we attempt to restart them
     _reset_failed_services()
     #FIXME: After config DB daemon is implemented, we'll no longer need to restart every service.
+    log_info("'load_minigraph' restart services...")
```
Comment (jleveque) [inline config/main.py]: Suggest: s/restart/restarting/

File: config/main.py [hunk 4]
```diff
@@ -681,6 +683,7 @@ def load_minigraph():
         device_type = device_type.strip()
 
     #Stop services before config push
+    log_info("'load_minigraph' stop services...")
```
Comment (jleveque) [inline config/main.py]: Suggest: s/stop/stopping/

File: config/main.py [hunk 5]
```diff
@@ -708,6 +711,7 @@ def load_minigraph():
     # status from all services before we attempt to restart them
     _reset_failed_services()
     #FIXME: After config DB daemon is implemented, we'll no longer need to restart every service.
+    log_info("'load_minigraph' restarting services...")
```
Comment (lguohan) [inline config/main.py]: restarting instead of starting.

**Pull request merged**